### PR TITLE
feat(remote-menu):option for disable mute and kick

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -466,6 +466,15 @@ var config = {
...
     // },
 
+    // Options related to the remote participant menu.
+    // remoteVideoMenu: {
+    //     // If set to true the 'Kick out' button will be disabled.
+    //     disableKick: true
+    // },
+
+    // If set to true all muting operations of remote participants will be disabled.
+    // disableRemoteMute: true,
```
Comment (saghul) [inline config.js]: Why is this not part of the remoteVideoMenu thing?

Comment (hristoterezov) [reply config.js]: Because on true, for web, the overflow menu item 'Mute everyone' will be also disabled.

Comment (saghul) [reply config.js]: Then let'ss have an option for every item we need to disable. As it stands I don't undderstand what this option does.

Comment (saghul) [reply config.js]: Yes, I see the value in that. But since those options are all inside the "remote video menu" we should have it there.

Comment (hristoterezov) [reply config.js]: @saghul I'm sorry if I'm missing something but as I said NOT all remote mute buttons are in the remote video menu. There is a 'Mute everyone' button in the secondary overflow menu which is part of the toolbox (see the screenshot). Let me know if I missed something!
Screenshot 2020-03-29 at 3 08 37 PM

Comment (saghul) [reply config.js]: Ah right, sorry my bad. That begs the question of course: why do we need this if there is already a way to do it with INTERFACE_CONFIG?

Comment (hristoterezov) [reply config.js]: I got really lost now! Sorry!

Why do we need what? Also what do we have in INTERFACE_CONFIG?

**Comment by doomtrooper** [comment]
@hristoterezov @saghul Can you please explain how will this work for Native `Android` or `iOS`? I don't see this exposed to Native SDKs through [Constants.js](https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/flags/constants.js). Please excuse me, I am still new to the `React` world.

**Pull request merged**